frontends/web/src/components/anchor/anchor.tsx

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import { h, RenderableProps } from 'preact';
+import { h, JSX, RenderableProps } from 'preact';
```

Comment:
TypeScript complained about returning JSX.Element, here https://github.com/digitalbitbox/bitbox-wallet-app/pull/1027/files#diff-c46c03dfb317b41884c6e334884a9bceR26

Ideally we wouldn't have to import JSX

Reply:
Why did it build successfully without importing it before?

Reply:
Correction: Typescript doesn't complain but typescript-eslint. Finally found one having same issue: https://github.com/typescript-eslint/typescript-eslint/issues/2477#issuecomment-686845188

I tried going down that rabit hole (using @types/react), but stopped now and postpone until we move to TS4 + PreactX.